tests/core/vm/test_state_access_restrictions.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+
+from evm.exceptions import UnannouncedStateAccess
+
+from evm.utils.numeric import (
+    big_endian_to_int,
+)
+from evm.utils.state_access_restriction import (
+    to_prefix_list_form,
+)
+
+from tests.core.fixtures import chain  # noqa: F401
```

Comment:
Note to self: TODO: clean this up to live in `tests.core.conftest` and remove the fixture imports from tests.